src/lib/temple/ledger-live.ts

```diff
@@ -1,11 +1,19 @@
+import { TransportType } from '@temple-wallet/ledger-bridge';
+
 import { TempleSharedStorageKey } from 'lib/temple/types';
 
 export async function isLedgerLiveEnabledByDefault() {
-  return process.env.TARGET_BROWSER === 'chrome';
+  return process.env.TARGET_BROWSER === 'chrome' ? TransportType.LEDGERLIVE : TransportType.U2F;
+}
+
+export function isLedgerLiveEnabled() {
```

Comment:
I recommend renaming this module name and its method names.

Reply:
For example:
`isLedgerLiveEnabledByDefault` -> `getDefaultLedgerTransport`
`isLedgerLiveEnabled ` -> `pickLedgerTransport`